### DISCUSSION TITLE
Inconsistent (?) number of goals in "Breakdown by"

### DISCUSSION THREAD

**Discussion by kleuter** [plausible/analytics#1269] [Anything Else]
Hi,

The "issue" is the same across all goals and all domains monitored.

![image](https://user-images.githubusercontent.com/2600624/130647091-5c22407a-8ce8-4cad-87de-e651953f5f69.png)

13.5k+5.3k = 18.8k (not 17.5k)

Is this normal? Thanks.

**Comment by ukutaht** [comment]
Hey!

It's common to see that the numbers in your dashboard aren't _'adding up'_.

This happens because visitors can view more than a single page or trigger more than a single custom event on your site. The easiest way to see this is to imagine a single visitor triggering two _Download_ events: once with `Format: ZIP` and once with `Format: 7Z`. In this case your Plausible dashboard would show 1 unique visitor in total. However, the breakdown report would show one unique visitor for `Format: ZIP` and one unique visitor for `Format: 7Z`.

Adding up unique visitor numbers and comparing them to the total is misleading. Since visitors aren't limited to a single pageview or action on your site, you shouldn't expect these numbers to be equal.

Hope that answers the question :)

**Comment by kleuter** [comment]
Thank you for the detailed explanation 👍
